resources/pvc.py

```diff
@@ -71,21 +71,21 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_pvcs(pvc_name):
```

Comment:
delete_pvc()
In case you want to delete multiple pvcs, please pass a list to this function

Reply:
should i check isinstance(pvc_name, str) if its str will delete single pvc if list will parse and delete

Reply:
Done delete_pvc will accept list of pvc for deletion